src/scripts/main.js

```diff
@@ -1,3 +1,34 @@
 'use strict';
 
-// write code here
+const listItems = [...document.querySelectorAll('li')];
+const employeesList = document.querySelector('ul');
+
+function convertToNumber(string) {
+  const salaryString = string;
+  const salaryNumber = salaryString.replace('$', '').replace(',', '');
+
+  return +salaryNumber;
+}
+
+function sortBySalary(items, list) {
+  items.sort(
+    (a, b) =>
+      convertToNumber(b.dataset.salary) - convertToNumber(a.dataset.salary),
+  );
+
+  list.innerHTML = '';
+  items.forEach((item) => list.appendChild(item));
+}
+
+function getEmployees(items) {
+  return items.map((item) => ({
+    name: item.textContent.trim(),
+    position: item.dataset.position,
+    salary: item.dataset.salary,
+    age: item.dataset.age,
+  }));
+}
```

Comment:
This function is fine, but it's not used anywhere. If you don't use the result of this function, it's better to remove it. Remember, unused code can lead to confusion and unnecessary complexity.